include/zephyr/drivers/dma/dma_stm32.h

```diff
@@ -12,6 +12,24 @@
  */
 #define STM32_DMA_HAL_OVERRIDE      0x7F
 
+/* @brief gives the first DMA channel : 0 or 1 in the register map
+ * when counting channels from 1 to N or from 0 to N-1
+ */
+#if defined(CONFIG_DMA_STM32_V3)
```

Comment:
`config DMA_STM32_V3` doesn't exist (yet in main branch)

Reply:
yes,